base/docs/basedocs.jl

```diff
@@ -2413,8 +2413,8 @@ julia> 4.5/2
 """
     ArgumentError(msg)
 
-The parameters to a function call do not match a valid signature. Argument `msg` is a
-descriptive error string.
+The arguments passed to a function are invalid.
+`msg` is a descriptive error string.
```

Comment:
Maybe, while you are at it:
```suggestion
`msg` is a descriptive error message.
```
(possibly with adding `::AbstractString` to the signature)
I don't find it clear what an  "error string" is...